src/hres_ic.py

```diff
@@ -0,0 +1,63 @@
+#!/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.01/bin/python
```

Comment:
Remove the explicit path to Python and replace it with (e.g.)
```
#!/bin/env python3
```
as appropriate.

Reply:
As above